webapp/pages/index.vue

```diff
@@ -10,7 +10,7 @@
         />
       </ds-flex-item>
       <hc-post-card
-        v-for="(post, index) in uniq(Post)"
+        v-for="(post, index) in posts"
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-07-09T21:49:06Z" title="Tuesday, July 9th 2019, 11:49:06 pm +02:00">Jul 9, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -10,7 +10,7 @@
         />
       </ds-flex-item>
       <hc-post-card
-        v-for="(post, index) in uniq(Post)"
+        v-for="(post, index) in posts"
```


maybe I need to add this `uniq` back??

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-09T23:52:35Z" title="Wednesday, July 10th 2019, 1:52:35 am +02:00">Jul 10, 2019</time>_

---


I dunno, what could add a duplicate post here?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-07-10T18:10:39Z" title="Wednesday, July 10th 2019, 8:10:39 pm +02:00">Jul 10, 2019</time>_

---


I don't know... I think @appinteractive told @Tirokk some reason to leave it in, but I'm not sure